src/me/coley/recaf/ui/component/editor/InsnListEditor.java

```diff
@@ -198,6 +193,55 @@ private void setupModel() {
 					getItems().removeAll(selectionModelProperty().getValue().getSelectedItems());
 				}
 			});
+			addEventHandler(KeyEvent.KEY_RELEASED,e -> {
+				ConfKeybinds keys = ConfKeybinds.instance();
+				if (keys.active && !e.isControlDown()) {
+					return;
+				}
+				if (e.getCode() != KeyCode.F) {
+					return;
+				}
+				Dialog<String> contentSearchDialog = new Dialog<>();
+				contentSearchDialog.setTitle(Lang.get("ui.bean.method.instructions.search.title"));
+				TextFlow textFlow = new TextFlow();
+				TextField textField = new TextField();
+				textField.addEventHandler(KeyEvent.KEY_RELEASED, event -> {
+					String textFieldText = textField.getText();
+					if (event.getCode()== KeyCode.ENTER) {
+						contentSearchDialog.setResult(textFieldText);
+					}
+				});
+				Button button = new Button();
+				button.setText(Lang.get("ui.bean.method.instructions.search.confirm"));
+				button.addEventHandler(MouseEvent.MOUSE_RELEASED,event -> {
+					if (event.getButton() != MouseButton.PRIMARY) {
+						return;
+					}
+					contentSearchDialog.setResult(textField.getText());
+				});
+				textFlow.getChildren().add(textField);
+				textFlow.getChildren().add(button);
+				contentSearchDialog.getDialogPane().setContent(textFlow);
+				try {
+					String result = contentSearchDialog.showAndWait().orElseThrow(NullPointerException::new);
+					if (result.isEmpty()) {
+						return;
+					}
+					getSelectionModel().clearSelection();
+					nodeLookup.entrySet()
+							.stream()
+							.filter(entry -> {
+								String text = entry.getValue().getText();
+								return text.contains(result);
+							})
+							.map(Map.Entry::getKey)
+							.forEach(abstractInsnNode -> {
+								getSelectionModel().select(abstractInsnNode);
+							});
+				} catch (NullPointerException e1) {
+					contentSearchDialog.close();
```

Comment:
Some feedback that no results have been found would be :+1: 
I used `Toolkit.getDefaultToolkit().beep();` in [`FxCode`](https://github.com/Col-E/Recaf/blob/master/src/me/coley/recaf/ui/FxCode.java)